webapp/app/templates/basis/base_step_form.html

```diff
@@ -42,7 +42,7 @@
     <script>
         $(document).ready(function(){
           $('.date_input').mask('00.00.0000');
-          $('input[id=iban]').mask('SS00 0000 0000 0000 0000 00', { reverse: true });
+          $('input[id=iban]').mask('AA00 0000 0000 0000 0000 00', { reverse: true });
```

Comment:
Wow, the documentation of that plugin is awful :(

What does `reverse: true` do?

Reply:
Ah! Got it, thanks.